implementations/rust/ockam/ockam_core/src/routing/address.rs

```diff
@@ -119,7 +123,10 @@ impl Address {
 
     /// Generate a random address with a specific type
     pub fn random(tt: u8) -> Self {
-        Self { tt, ..random() }
```

Comment:
How is new implementation different from old one?

Reply:
Nice catch - I wrote that before I implemented `random()` for `no_std` - I'll revert it.